js/app/demo/configuration.jsx

```diff
@@ -6,10 +6,27 @@ define(["react", "jsx!parserOptions", "jsx!environments", "jsx!rulesConfig"], fu
         getInitialState: function() {
             return { showingConfig: false };
         },
+        initPopovers: function() {
+            var popoverElements = document.querySelectorAll("[data-toggle=\"popover\"]");
+
+            for (var i = 0; i < popoverElements.length; ++i) {
+                // eslint-disable-next-line no-new, no-undef
```

Comment:
Why are you seeing `no-undef` errors here?

Is it due to the `Popover` reference? If so, the better way to handle that is to add Popover as a global, either to the main ESLint config or as a comment near the top of the file. See [configuration guide](https://eslint.org/docs/user-guide/configuring#specifying-globals) for more info.